activemodel/lib/active_model/serialization.rb

```diff
@@ -72,10 +72,20 @@ module ActiveModel
   module Serialization
     # Returns a serialized hash of your object.
     #
+    #   class Address
+    #     include ActiveModel::Serialization
+    #
+    #     attr_accessor :city, :street
+    #
+    #     def attributes
+    #       {'city' => nil, 'street' => nil}
```

Comment:
Not according to all the rest of the documentation in this class. All the other examples just use `=> nil` everywhere.

In fact, the values are completely irrelevant. `attributes` is used in only [one place](https://github.com/rails/rails/blob/master/activemodel/lib/active_model/serialization.rb#L100) and that is only to extract the keys.

As far as I can tell, the only reason a Hash is expected here instead of an array of keys is so it would have the same requirements as `ActiveModel::AttributeMethods`.